### DISCUSSION TITLE
同步笔记的文件夹换了地方，能不能把旧的笔记批量同步到新的文件夹里面？I have moved the folder where my synced notes are stored. Is it possible to batch sync the old notes to the new folder?

### DISCUSSION THREAD

**Discussion by a1576799** [windingwind/zotero-better-notes#1046] [Q&A]
同步笔记的文件夹换了地方，能不能把旧的笔记批量同步到新的文件夹里面？

I have moved the folder where my synced notes are stored. Is it possible to batch sync the old notes to the new folder?

**Comment by windingwind** [comment]
Tools - Sync Manager -Detect...

**Reply by a1576799** [reply]
![image](https://github.com/user-attachments/assets/23a88c42-4f97-4471-bdb5-2fb59ab789a9)

I didnt see the detect button

**Reply by windingwind** [reply]
Update plugin
